packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -52,5 +54,6 @@ data class GutenbergProps(
         const val PROP_CAPABILITIES = "capabilities"
         const val PROP_CAPABILITIES_MENTIONS = "mentions"
         const val PROP_CAPABILITIES_UNSUPPORTED_BLOCK_EDITOR = "unsupportedBlockEditor"
+        const val PROP_NAME_MODAL_LAYOUT_PICKER = "modalLayoutPicker"
```

Comment:
Not a big deal, but for the sake of consistency with the other "capabilities" props, what would you think about renaming this variable to "PROP_CAPABILITIES_MODAL_LAYOUT_PICKER"?

Reply:
You are right @mchowning . I copy pasted the wrong naming from the old code.
Thank you for your feedback :)